### ISSUE TITLE
Can't reproduce the same nice result for Example1

### ISSUE THREAD

**Issue by hhxjzyr** [anishathalye/neural-style#105]
After reading the code and readme in this repo, I tried to reproduce the nice result in the readme Example 1 

with the simplest command and with all  the default parameters as the example 1 told us:
`python neural_style.py --content examples/1-content.jpg --styles examples/1-style.jpg --output output.jpg` 

 **default CONTENT_WEIGHT = 5e0 default  STYLE_WEIGHT = 5e2 default ITERATIONS = 1000** , same as exmple 1 used

but I got a very bad result.... and here is it

![image](https://user-images.githubusercontent.com/7722295/33163437-7a80a7e4-d069-11e7-903e-386683f0b954.png)

I am using 

system & hardware:
- Ubuntu16.04 
- CUDA 8.0 
- cudnn-8.0-linux-x64-v6.0
- two tesla k40m

python dependencies:
- tensorflow-gpu 1.3.0
- numpy 1.13.3
- pillow  4.2.1
- scipy  0.19.1

**Comment by anishathalye** [comment]
How many iterations did you run it for?

**Comment by anishathalye** [comment]
I just checked out commit b7461815c8fee73199709273c76daf69982db3ec and ran:

```bash
python neural_style.py --content examples/1-content.jpg --styles examples/1-style.jpg --output output.jpg
```

On completion, the resulting losses were:

``` 
  content loss: 753232
    style loss: 249069
       tv loss: 51529.7
    total loss: 1.05383e+06
```

And the resulting image was:

![output](https://user-images.githubusercontent.com/3526486/33179761-897e075e-d038-11e7-8dee-606e8561d950.jpg)

**Comment by hhxjzyr** [comment]
I found that it is because my optimization process wasn't going well.......
And I tried to cut down my learning rate from 1 to 0.5 but it still didn't work.
The **content loss** is exploding.... the **style loss** is dropping as we expected.
But I used the code at commit b746181 exactly...
`
Optimization started...
  content loss: 2.13976e+06
    style loss: 1.69467e+10
       tv loss: 26.2307
    total loss: 1.69488e+10
Iteration    1/1000
  content loss: 2.04499e+06
    style loss: 1.68075e+10
       tv loss: 23626.3
    total loss: 1.68096e+10
Iteration    2/1000
  content loss: 2.14448e+06
    style loss: 1.62887e+10
       tv loss: 51806.2
    total loss: 1.62909e+10
Iteration    3/1000
  content loss: 2.71498e+06
    style loss: 1.53005e+10
       tv loss: 87189
    total loss: 1.53033e+10
Iteration    4/1000
  content loss: 3.94074e+06
    style loss: 1.38743e+10
       tv loss: 133313
    total loss: 1.38784e+10
Iteration    5/1000
  content loss: 5.92366e+06
    style loss: 1.21259e+10
       tv loss: 193598
    total loss: 1.2132e+10
Iteration    6/1000
  content loss: 8.7247e+06
    style loss: 1.02663e+10
       tv loss: 270708
    total loss: 1.02753e+10
Iteration    7/1000
  content loss: 1.23115e+07
    style loss: 8.62048e+09
       tv loss: 366224
    total loss: 8.63315e+09
Iteration    8/1000
  content loss: 1.63399e+07
    style loss: 7.49406e+09
       tv loss: 477425
    total loss: 7.51088e+09
Iteration    9/1000
  content loss: 1.98463e+07
    style loss: 6.81471e+09
       tv loss: 592838
    total loss: 6.83515e+09
Iteration   10/1000
  content loss: 2.18546e+07
    style loss: 6.17265e+09
       tv loss: 700191
    total loss: 6.19521e+09
Iteration   11/1000
  content loss: 2.25083e+07
    style loss: 5.53644e+09
       tv loss: 797079
    total loss: 5.55975e+09
Iteration   12/1000
  content loss: 2.28152e+07
    style loss: 5.07227e+09
       tv loss: 884799
    total loss: 5.09597e+09
Iteration   13/1000
  content loss: 2.35705e+07
    style loss: 4.69006e+09
       tv loss: 963357
    total loss: 4.71459e+09
Iteration   14/1000
  content loss: 2.48656e+07
    style loss: 4.27918e+09
       tv loss: 1.03338e+06
    total loss: 4.30508e+09
Iteration   15/1000
  content loss: 2.6452e+07
    style loss: 3.85858e+09
       tv loss: 1.09631e+06
    total loss: 3.88612e+09
Iteration   16/1000
  content loss: 2.80181e+07
    style loss: 3.48507e+09
       tv loss: 1.1535e+06
    total loss: 3.51425e+09
Iteration   17/1000
  content loss: 2.92717e+07
    style loss: 3.18076e+09
       tv loss: 1.20593e+06
    total loss: 3.21124e+09
Iteration   18/1000
  content loss: 3.00439e+07
    style loss: 2.93077e+09
       tv loss: 1.25453e+06
    total loss: 2.96207e+09
Iteration   19/1000
  content loss: 3.03715e+07
    style loss: 2.71745e+09
       tv loss: 1.30068e+06
    total loss: 2.74912e+09
Iteration   20/1000
  content loss: 3.04512e+07
    style loss: 2.53742e+09
       tv loss: 1.34613e+06
    total loss: 2.56922e+09
Iteration   21/1000
  content loss: 3.05081e+07
    style loss: 2.38572e+09
       tv loss: 1.39239e+06
    total loss: 2.41762e+09
Iteration   22/1000
  content loss: 3.06965e+07
    style loss: 2.25052e+09
       tv loss: 1.44025e+06
    total loss: 2.28266e+09
Iteration   23/1000
  content loss: 3.10736e+07
    style loss: 2.12281e+09
       tv loss: 1.48963e+06
    total loss: 2.15537e+09
Iteration   24/1000
  content loss: 3.15917e+07
    style loss: 2.00316e+09
       tv loss: 1.53984e+06
    total loss: 2.03629e+09
Iteration   25/1000
  content loss: 3.21356e+07
    style loss: 1.8944e+09
       tv loss: 1.58965e+06
    total loss: 1.92813e+09
Iteration   26/1000
  content loss: 3.25785e+07
    style loss: 1.79537e+09
       tv loss: 1.63772e+06
    total loss: 1.82958e+09
Iteration   27/1000
  content loss: 3.28324e+07
    style loss: 1.70156e+09
       tv loss: 1.68294e+06
    total loss: 1.73608e+09
Iteration   28/1000
  content loss: 3.28908e+07
    style loss: 1.61048e+09
       tv loss: 1.72478e+06
    total loss: 1.64509e+09
...



`

**Comment by hhxjzyr** [comment]
After I upgraded my tensorflow-gpu from version `1.3` to version `1.4`. Everything went ok...

So I don't know the reason.........

**Comment by anishathalye** [comment]
Strange. Glad it's working now though!
